gemini/use-cases/retrieval-augmented-generation/utils/intro_multimodal_rag_utils.py

```diff
@@ -131,6 +115,9 @@ def get_image_embedding_from_multimodal_embedding_model(
     )
```

Comment:
Can you update this function to use the Vertex AI SDK? You should be able to change it doing something like this:

```
image_embedding = (
    multi_modal_embedding_model.get_embeddings(
        image=embeddings_image,
        embedding_size=embedding_size,
    ).image_embedding,
)

image_description_text_embedding = (
    multi_modal_embedding_model.get_embeddings(
        contextual_text=response[:text_emb_text_limit],
        embedding_size=embedding_size,
    ).text_embedding,
)
```

I updated the SDK to allow specific dimensions (which it didn't before)

https://cloud.google.com/python/docs/reference/aiplatform/latest/vertexai.vision_models.MultiModalEmbeddingModel#vertexai_vision_models_MultiModalEmbeddingModel_get_embeddings

The docs haven't updated yet.

Reply:
Changed to "MultiModalEmbeddingModel" class using: 

from vertexai.vision_models import MultiModalEmbeddingModel

Retested the notebook and outputs are not affected.